README.md

```diff
@@ -73,7 +83,7 @@ node 'default' {
 }
 ~~~
 
-The preceding configuration will cause the next `puppet agent` run to automatically download the current `cisco_node_utils` gem from <https://rubygems.org/gems/cisco_node_utils> and install it on the node.
+The preceding configuration will cause the next `puppet agent` run to automatically download the current `cisco_node_utils` and `puppet-resource_api` gems from <https://rubygems.org/> and install them on the node.

```

Comment:
Created https://tickets.puppetlabs.com/browse/FM-7624 to create the correct classes that should be described here. 

Customers should never do what is currently described here.